src/data/VideoFile.py

```diff
@@ -92,11 +93,23 @@ def next(self) -> np.ndarray:
             ret, frame = self.capture.read()
             if ret:
                 self.current_frame = frame
+                self.current_time = self.capture.get(cv2.CAP_PROP_POS_MSEC)
 
         return self.current_frame
 
     def current(self):
         return self.current_frame
+    
+    def current_time(self):
+        return self.current_time
```

Comment:
does this work? we have the function current_time and the variable which have the same name?

Reply:
I agree it's confusing. I will just keep the field